simulator-core/src/test/java/com/evbox/everon/ocpp/it/security/UpdateChargingStationCertificateByRequestOfCSMSIt.java

```diff
@@ -0,0 +1,80 @@
+            "308203643082024c02021235300d06092a864886f70d0101050500307a310b3009060355040613024e4c310b300906035504080c024e4c310c300a06035504070c03414d53310f300d060355040a0c06526f6f744341310f300d060355040b0c06526f6f744341310f300d06035504030c06526f6f744341311d301b06092a864886f70d010901160e726f6f74636140726f6f742e6361301e170d3139313032313133303633365a170d3230313032303133303633365a3075310b3009060355040613024e4c310b300906035504080c024e4c310c300a06035504070c03414d53310e300c060355040a0c055375624341310e300c060355040b0c055375624341310e300c06035504030c055375624341311b301906092a864886f70d010901160c7375626361407375622e636130820122300d06092a864886f70d01010105000382010f003082010a0282010100b8025e34f372a4957af6b12ebc07475770c8dd1a9b657eeb5e9f1790b8217c3652feef703c4407895d740658a68ce26d890266d27d8115d587f7ee011aa14bd56494cadf81fecf20ba67ace4e96154b0b61a0860a0818a1d65631533e61f07d5857a6d33640e87273d5ee4b5446b55e15fb2dc1f4416ae13a07f64560ba6d51a6dfa52b7dd5f24039a7b8094c84ddb36ae7c037dc306df5b6a914a0409102611ecf0a3a5e390620fb9d6ec0ad48a1ccaf61c0257bebe6186edabeaba9bd3966f69f20a49190cf6dda28d1367fe66882aafed27fdf01bcc8adc8b629048148b84fb76cdbfc68ba5f9736090779160725be68db41822c480a7f3f0c7ea3acc27c70203010001300d06092a864886f70d010105050003820101007dacf7835ca9865f0f146acba45c97ce71386c7afc769f8f05fff59c11e813d7ca63904b1bafba386791f913b6b1be17bdadf2c23438aeb6c1a04bba9a69e0c7c7dd14db0d6c21e395e0aedd310fba5bb3746779cb0a6d4ad64979023fccf170649ac35b2ade0d199f75e6d2e2c1b43c49976298b2d79f5e1004e2cb40274110bde4ebe44f03edd5b6f8707f99ba17038f7971af8d1da21238b014eeb819580cfc33bfc4328050c62589023c79631f82925802a080ab4338fcc199283c924db682806c8cb2d5cd2834bb10a50633d6a0eae0b05ad74c7b2e51b50b06df7955528411f28b1c1f82c469b2b8e230ca63875091d8b64e387132267f66dd18bee54f";
+
+    private static final String pemCertificate = "-----BEGIN CERTIFICATE-----" +
```

Comment:
can we put it to a resource file? Anyway it's not human readable. The string above as well